xbmc/settings/AdvancedSettings.cpp

```diff
@@ -109,7 +109,7 @@ void CAdvancedSettings::Initialize()
 
   m_audioHeadRoom = 0;
   m_ac3Gain = 12.0f;
-  m_audioApplyDrc = true;
+  m_audioApplyDrc = -1.0f;
```

Comment:
How - those values are really hard to communicate - also this is ffmpeg ac3 specific and most likely can't be used for anything else.